### PR TITLE
PS-8212 Add dates to the Release Note Index

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: source/release-notes/release-notes_index.rst

### CODE REVIEW

**Review by patrickbirch — COMMENTED** [review]
Dates are off - check all of them again
MySQL 5.7.10-1 on December 14th, 2015
Percona Server for MySQL 5.7.10-2 on February 5th, 2016
Percona Server for MySQL 5.7.10-3 on February 23rd, 2016
Percona Server for MySQL 5.7.11-4 on March 15th, 2016

File: doc/source/release-notes/release-notes_index.rst [hunk 1]
```diff
@@ -1,45 +1,82 @@
...
+
+   Percona Server 5.7.14-7 (2016-08-23) <Percona-Server-5.7.14-7.rst>
+
+   Percona Server 5.7.13-6 (2016-07-16) <Percona-Server-5.7.13-6.rst>
+
+   Percona Server 5.7.12-5 (2016-06-06) <Percona-Server-5.7.12-5.rst>
+
+   Percona Server 5.7.11-4 (2016-03-16) <Percona-Server-5.7.11-4.rst>
+
+   Percona Server 5.7.10-3 (2016-02-16) <Percona-Server-5.7.10-3.rst>
```
Comment (patrickbirch) [inline doc/source/release-notes/release-notes_index.rst]: Percona Server for MySQL 5.7.10-3 on February 23rd, 2016

File: doc/source/release-notes/release-notes_index.rst [hunk 2]
```diff
@@ -1,45 +1,82 @@
...
+
+   Percona Server 5.7.14-8 (2016-09-21) <Percona-Server-5.7.14-8.rst>
+
+   Percona Server 5.7.14-7 (2016-08-23) <Percona-Server-5.7.14-7.rst>
+
+   Percona Server 5.7.13-6 (2016-07-16) <Percona-Server-5.7.13-6.rst>
+
+   Percona Server 5.7.12-5 (2016-06-06) <Percona-Server-5.7.12-5.rst>
+
+   Percona Server 5.7.11-4 (2016-03-16) <Percona-Server-5.7.11-4.rst>
```
Comment (patrickbirch) [inline doc/source/release-notes/release-notes_index.rst]: Percona Server for MySQL 5.7.11-4 on March 15th, 2016

**Review by patrickbirch — COMMENTED** [review]
Check the dates

File: doc/source/release-notes/release-notes_index.rst (hunk 1 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/release-notes_index.rst]: Fixed

File: doc/source/release-notes/release-notes_index.rst (hunk 2 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/release-notes_index.rst]: Fixed

File: doc/source/release-notes/release-notes_index.rst [hunk 3]
```diff
@@ -1,45 +1,82 @@
...
+
+   Percona Server for MySQL 5.7.23-24 (2018-11-09) <Percona-Server-5.7.23-24.rst>
+
+   Percona Server for MySQL 5.7.23-23 (2018-09-12) <Percona-Server-5.7.23-23.rst>
+
+   Percona Server for MySQL 5.7.22-22 (2018-05-31) <Percona-Server-5.7.22-22.rst>
+
+   Percona Server for MySQL 5.7.21-21 (2018-04-24) <Percona-Server-5.7.21-21.rst>
+
+   Percona Server for MySQL 5.7.21-20 (2019-02-18) <Percona-Server-5.7.21-20.rst>
```
Comment (patrickbirch) [inline doc/source/release-notes/release-notes_index.rst]: Percona Server 5.7.21-20 on February 19, 2018

File: doc/source/release-notes/release-notes_index.rst [hunk 4]
```diff
@@ -1,45 +1,82 @@
...
-   Percona-Server-5.7.15-9
-   Percona-Server-5.7.14-8
-   Percona-Server-5.7.14-7
-   Percona-Server-5.7.13-6
-   Percona-Server-5.7.12-5
-   Percona-Server-5.7.11-4
-   Percona-Server-5.7.10-3
-   Percona-Server-5.7.10-2
-   Percona-Server-5.7.10-1
+   Percona Server for MySQL 5.7.35-38 (2021-08-18) <Percona-Server-5.7.35-38.rst>
```
Comment (patrickbirch) [inline doc/source/release-notes/release-notes_index.rst:12]: Percona Server for MySQL 5.7.37-40
2022-03-31
Percona Server for MySQL 5.7.36-39
Date:	December 22, 2021

Comment (alina-derkach-oaza) [reply doc/source/release-notes/release-notes_index.rst:12]: fixed

File: doc/source/release-notes/release-notes_index.rst (hunk 3 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/release-notes_index.rst]: fixed

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**